nginx/containerbuddy.json

```diff
@@ -0,0 +1 @@
+{"consul":"${CONSUL}:8500","onStart":"/opt/containerbuddy/reload.sh preStart","services":[{"name":"nginx","port": 80,"publicIp": true,"health":"/usr/bin/curl --fail -s http://localhost/health.txt","poll":10,"ttl":25}],"backends":[{"name":"wordpress","poll":5,"onChange":"/opt/containerbuddy/reload.sh"}]}
```

Comment:
that "preStart" is the argument to the reload.sh script to run the preStart function
https://github.com/autopilotpattern/nginx/blob/master/bin/reload.sh#L8-L14